client/src/pages/SelectVenue/spendPartnersPublicReducer.js

```diff
@@ -0,0 +1,34 @@
+import {
+  GET_SPEND_PARTNERS_FAIL,
+  GET_SPEND_PARTNERS_SUCCESS,
+  GET_SPEND_PARTNERS_LOADING,
+} from "../../redux/types";
+
+const initState = { spendPartners: [] };
```

Comment:
I'm sorry I just forgot to mention this but maybe you could rename it to data? so you'd call it `state.spendPartners.data` rather than `state.spendPartnersPublic.spendPartners`
```suggestion
const initState ={ loading: false,  error: null, data: [] };
```